src/lpc.jl

```diff
@@ -0,0 +1,75 @@
+"""
+`lpc_burg(x::Vector, p::Int)`
+
+LPC (Linear-Predictive-Code) estimation, using the Burg method.
+
+Given input signal `x` and prediction order `p`, returns IIR coefficients `a`
+and average reconstruction error `prediction_err`. Note that this method does
+NOT return the leading `1` present in the true autocorrelative estimate; it
+omits it as it is implicit in every LPC estimate, and must be manually
+reintroduced if the returned vector should be treated as a polynomial. This
+function implements the mathematics published in [1].
+
+[1] - Enhanced Partial Tracking Using Linear Prediction
+(DAFX 2003 article, Lagrange et al)
+http://www.sylvain-marchand.info/Publications/dafx03.pdf
+"""
+function lpc_burg{T <: Number}(x::Vector{T}, p::Int)
```

Comment:
A better API would probably have a parameter of type maybe `LPCBurg` (similar for levinson) and rename the function to `lpc`. Then the default would be `lpc(x, p) = lpc(x,p, LPCBurg())`.